README.md

```
@@ -17,551 +17,66 @@ DNS Synchronization Service for BIND9
 ```bash
-# Start bindizr on foreground
-$ bindizr start
-
-# Start with a custom configuration file
-$ bindizr start -c <FILE>
-
-# Check the current status of bindizr service
-$ bindizr status
-
-# Verify the installation end to end (config, daemon, API, database, DNS, secondaries)
 $ bindizr doctor
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)